pytket/extensions/quantinuum/backends/api_wrappers.py

```diff
@@ -469,3 +470,130 @@ def cancel(self, job_id: str) -> dict:
         jr = res.json()
 
         return jr  # type: ignore
+
+
+class QuantinuumAPIOffline:
+    def __init__(
+        self,
+        machine_list: Optional[list] = [
```

Comment:
```suggestion
        machine_list: list = [
```
No need for `Optional` I think?

Reply:
Mypy wanted it this way, might be a chain of other conditions, will try to find a different way.

Reply:
I think the real answer is Optional[list] = None?

If you give it a non-optional value I think you get that as the default?

Reply:
If you construct the Quantinuum backend you can decide if you want to hand over the QuantinuumAPI (default) or the new QuantinuumAPIOffline which removes the calls to the remote API. It is currently not possible to switch between them within a created backend, is that something you would like to do?